scripts/release/deploy_staging_jars.sh

```diff
@@ -38,7 +38,8 @@ fi
 
 declare -a ALL_VERSION_OPTS=(
 "-Dscala-2.11 -Dspark2.4 -Dflink1.13"
-"-Dscala-2.12 -Dspark2.4 -Dflink1.13"
+"-Dscala-2.11 -Dspark2.4 -Dflink1.14"
+"-Dscala-2.12 -Dspark2.4 -Dflink1.14"
 "-Dscala-2.12 -Dspark3.1 -Dflink1.14"
 "-Dscala-2.12 -Dspark3.2 -Dflink1.14"
```

Comment:
This is for published artifacts; there will be 
```
hudi-flink1.13-bundle_2.11
hudi-flink1.14-bundle_2.11
hudi-flink1.14-bundle_2.12
```
but no `hudi-flink1.13-bundle_2.12`

@danny0405 Given in previous releases we already published 2.12 bundles for flink, best if we continue support 2.12 for both flink profiles?

Reply:
Yes, that's true, both 2.11 and 2.12 profile are needed.